platform-core-module/src/main/resources/com/softicar/platform/core/module/page-style.css

```diff
@@ -78,13 +77,12 @@
 	left: 0px;
 }
 
```

Comment:
I understand that we have to reserve space on the top right for the working indicator, but it looks strange that the user profile button in the header bar is not completely right aligned. Maybe we could reduce the size of the working indicator and put it more to the top right, or show it over the page content div?

Reply:
Could you create a simple mockup of your idea?
...and can we do that in a separate PR? Because this PR did not change the layout (note that the user name was there before; it's just clickable now).